### ISSUE TITLE
How to train retinanet with normal Crossentroy !

### ISSUE THREAD

**Issue by banxia1994** [open-mmlab/mmdetection#2049]
I set use_sigmoid and use_focal_loss  to False, results for coco are low.

any suggestions?

**Comment by ZwwWayne** [comment]
It is quite normal if you do not use focal loss but does not change anything else. You need to add some sampling strategies or use different initialization methods. This work implemented by this PR https://github.com/open-mmlab/mmdetection/pull/1907 might help you.

**Comment by banxia1994** [comment]
actually，I already add random sampler .  according to origin focal loss paper， crossentry baseline is simple class-balanced， add α to pos and （1-α）to neg （all samples without using sampler）， related experienced is processing， other setting follow mmdet origin retinanet， Any comments on this modification? 

Thx a lot!

**Comment by ZwwWayne** [comment]
Since RetinaNet is not very stable, you might need to run the experiments several times.
I am afraid that I have no more comments now.

**Comment by rydenisbak** [comment]
@banxia1994 for Cross entropy common practice is using online hard negative mining.
You can take this code 
https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/anchor_heads/ssd_head.py#L113
and move it to retina_head.py and use it if loss_type is CrossEntropy otherwise use ordinady code

**Comment by banxia1994** [comment]
> @banxia1994 for Cross entropy common practice is using online hard negative mining.
> You can take this code
> https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/anchor_heads/ssd_head.py#L113
> and move it to retina_head.py and use it if loss_type is CrossEntropy otherwise use ordinady code

thanks！！
